types/p-log/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for p-log 1.0
+// Project: https://github.com/sindresorhus/p-log#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = pLog;
+
+declare const pLog: pLog;
+
+interface pLog {
+    <T>(): (value: T) => Promise<T>;
```

Comment:
Tried it out, this way I can't pass a type parameter to the function like I do in the tests.